platform-sdk/swirlds-platform-core/src/testFixtures/java/com/swirlds/platform/test/fixtures/event/source/AbstractEventSource.java

```diff
@@ -228,10 +229,12 @@ public IndexedEvent generateEvent(
         // work for consensus, and we will have to upgrade our simulation framework prior to converting
         // consensus to use the birth round.
         final IndexedEvent latestSelfEvent = getLatestEvent(random);
-        final long birthRound = Math.max(
-                        otherParentEvent == null ? 0 : otherParentEvent.getGeneration(),
-                        latestSelfEvent == null ? 0 : latestSelfEvent.getGeneration())
+        final long generation = Math.max(
+                        otherParentEvent == null ? (FIRST_GENERATION - 1) : otherParentEvent.getGeneration(),
```

Comment:
I think `GraphGenerations` as a source of constants is deprecated.  `EventConstants.FIRST_GENERATION` would be better.